### PR TITLE
Fix problem with the export dropdown

### PR DESCRIPTION
**Pull request by maximest-pierre**
*Fixes # .*

### Changes proposed in this pull request:

* Fix the formatting and javascript
* ...
* ...

### Status

- [x] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### Deployment notes and migration

- [ ] Migration needed

Fill me in ...

@savoirfairelinux/mll

There was a missing function in javascript to use the dropdown.

### CODE REVIEW

**Comment by manumilou** [comment]
The export does not take the queryset filtered, as it should.

**Comment by manumilou** [comment]
@maximest-pierre Still stuck on this one?

**Comment by maximest-pierre** [comment]
Yep

File: django/santropolFeast/santropolFeast/templates/system/_button_export.html [hunk 1]
```diff
@@ -2,10 +2,6 @@
     <div class="ui button">Export</div>
     <div class="ui floating dropdown icon button" tabindex="0">
         <i class="dropdown icon"></i>
-        <div class="menu transition hidden" tabindex="-1">
-            <div class="item">
-                <i class="dropdown icon"></i>
-                <span class="text">Export</span>
                 <div class="menu transition hidden">
                     <div class="header">Format</div>
                     <div class="item"><a href="?format=csv">CSV</a></div>
```
Comment (manumilou) [inline django/santropolFeast/santropolFeast/templates/system/_button_export.html]: You need this: 
`<div class="item"><a href="{{request.get_full_path}}&format=csv">CSV</a></div>`

**Comment by manumilou** [comment]
@maximest-pierre Can you fix the PEP8 error?

**Comment by maximest-pierre** [comment]
Yeah I am going to fix and rebase the whole commit.

**Pull request merged**